### PR TITLE
style and efficiency improvements

### PR DESCRIPTION
**Pull request by xiaozhou**

### CODE REVIEW

File: contracts/lib/Govern.sol [hunk 1]
```diff
@@ -132,9 +132,9 @@ contract Govern is IGovern, Ownable {
      * @param _record the key of this parameter
      * @param _value the new proposed value of this parameter
      */
-    function createParamProposal(uint256 _record, uint256 _value) public {
+    function createParamProposal(uint256 _record, uint256 _value) external {
         ParamProposal storage p = paramProposals[nextParamProposalId];
-        nextParamProposalId = nextParamProposalId.add(1);
+        nextParamProposalId = nextParamProposalId + 1;
```
Comment (Xuefeng-Zhu) [inline contracts/lib/Govern.sol:137]: for efficiency?

Comment (xiaozhou) [reply contracts/lib/Govern.sol:137]: yes

File: contracts/DPoS.sol [hunk 2]
```diff
@@ -464,7 +489,7 @@ contract DPoS is IDPoS, Ownable, Pausable, WhitelistedRole, Govern {
     function withdrawFromUnbondedCandidate(address _candidateAddr, uint256 _amount)
         external
         onlyNonZeroAddr(_candidateAddr)
-        minAmount(_amount, 1 ether)
+        minAmount(_amount, 1 * DECIMALS_MULTIPLIER)
```
Comment (Xuefeng-Zhu) [inline contracts/DPoS.sol:492]: Why prefer DECIMALS_MULTIPLIER  here? Make the code easier to understand?

Comment (xiaozhou) [reply contracts/DPoS.sol:492]: yes, avoid confusion, as it is CELR not ether,

**Review by Xuefeng-Zhu — APPROVED** [review]

**Review by Dominator008 — APPROVED** [review]

**Pull request merged**